examples/tutorials/advanced/Federated Recurrent Neural Network.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Federated Learning of a Recurrent Neural Network for text classification"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, you are going to learn how to train a Recurrent Neural Network (RNN) in a federated way with the purpose of *classifying* a person's surname to its most likely language of origin. \n",
+    "\n",
+    "\n",
+    "We will train two Recurrent Neural Networks residing on two remote workers based on a dataset containing approximately 20.000 surnames from 18 languages of origin, and predict to which language a name belongs based on the name's spelling. \n",
+    "\n",
+    "A **character-level RNN** treats words as a series of characters - outputting a prediction and “hidden state” per character, feeding its previous hidden state into each next step. We take the final prediction to be the output, i.e. which class the word belongs to. Hence the training process proceeds sequentially character-by-character\n",
+    "\n",
+    "Following distributed training, the resulting models are going to be able to perform operations like the following ones:"
+   ]
+  },
+  {
+   "cell_type": "raw",
```

Comment:
I don't know why/how, but there is text leaving the first cell:

![image](https://user-images.githubusercontent.com/8157164/57474748-b937b580-728a-11e9-8aa5-516802c08b83.png)

I think is because this cell has type "raw" instead of "markdown", can you change it to markdown?

Reply:
I have this same issue